charts/consul/templates/client-snapshot-agent-deployment.yaml

```diff
@@ -229,6 +246,7 @@ spec:
               {{- if .Values.global.adminPartitions.enabled }}
               -partition={{ .Values.global.adminPartitions.name }} \
               {{- end }}
+              -token-sink-file=/consul/login/acl-token \
```

Comment:
consul login will save the acl token to this sink file location.
snapshot agent needs an acl token and if not passed on through the `-token` flag, it will look `CONSUL_HTTP_TOKEN_FILE`...which is mapped to this location in that container.

Reply:
This is also the default in the `acl-init ` command but I think it's good to set this explicitly:

https://github.com/hashicorp/consul-k8s/blob/acls-refactor-base-branch/control-plane/subcommand/acl-init/command.go#L30